doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -1772,6 +1776,12 @@ \subsubsection{Outputs}\label{outputs-4-010}
 
 These outputs are the heat transfer rate and total heat transfer due to false loading of the chiller. When the chiller part-load ratio is below the minimum unloading ratio, the chiller false loads (e.g.~hot-gas bypass) to further reduce capacity. The false load heat transfer output variable is not metered.
 
+\paragraph{Chiller Evaporative Condenser Water Volume {[}m3{]}}\label{chiller-evaporative-condenser-water-volume-m3}
+
+\paragraph{Chiller Evaporative Condenser Mains Supply Water Volume {[}m3{]}}\label{chiller-evaporative-condenser-mains-supply-water-volume-m3}
+
+These outputs are the water use for evaporatively-cooled condensers. When the chiller operates, the water consumed by the evaporatively-cooled condenser is proportional to the chiller condenser heat transfer. The evaporative condenser is assumed to be 100% effective where the condenser inlet air dry-bulb temperature is equal to the outdoor wet-bulb temperature. 
+
```

Comment:
So the dimmed characters on my screen were real. I guess the % needs some type of delimiter. I see \% on Google.